courses/free-courses-en.md

```diff
@@ -275,6 +275,10 @@
 
 ### Computer Organization and Architecture
 
+* [Computer Architecture Complete course Part 1](https://youtu.be/9nuAjYRbITQ) - Nerd's lesson (YouTube, Video Length 9:29:50)
+* [Computer Architecture Complete course Part 2](https://youtu.be/foCkCSZXOt8) - Nerd's lesson (YouTube, Video Length 9:29:50)
+* [Computer Architecture Complete course Part 3](https://youtu.be/BHMETwOXvWw) - Nerd's lesson (YouTube, Video Length 9:29:50)
+* [Computer Organization & Architecture](https://www.learncomputerscienceonline.com/computer-organization-and-architecture) - Learn Computer Science
```

Comment:
Please,

- Don't invent titles. Use the one provided by the original source.
- For YT we prefer playlist URLs. Don't shorten. If are videos, we prefer those > 1h with chapters integrated in timeline.
- Don't invent new notes, e.g. timing length
- Also YT videos/playlist don't need annotations

Fixing...

```suggestion
* [Computer Architecture Complete course Part 1](https://www.youtube.com/watch?v=9nuAjYRbITQ) - Nerd's lesson
* [Computer Architecture Complete Course Part 2](https://www.youtube.com/watch?v=foCkCSZXOt8) - Nerd's lesson
* [Computer Architecture Complete course Part 3](https://www.youtube.com/watch?v=BHMETwOXvWw) - Nerd's lesson
* [Computer Organization And Architecture \| COA Tutorial With Examples](https://www.learncomputerscienceonline.com/computer-organization-and-architecture) - Learn Computer Science
```